cli/internal/hashing/package_deps_hash.go

```diff
@@ -99,10 +115,19 @@ func GetHashableDeps(rootPath fs.AbsolutePath, files []turbopath.AbsoluteSystemP
 		}
 		output[index] = anchoredSystemPath
 	}
-	return gitHashObject(convertedRootPath, output)
+	hashObject, err := gitHashObject(convertedRootPath, output)
```

Comment:
Can you add a TODO here that we should detect `git` once, rather than fail and fallback on each file?

Reply:
This is operating on an array of files, not individual files - so we throw the array at the git hash, if it fails we do it manually. 

We are assuming `git` first, instead of doing a check for git first, and then using the appropriate hashing function for the files array (which we could do as an optimization)

Reply:
Gotcha